### PR TITLE
[Filebeat[ Fix saml status code parsing for google workspace

### PR DESCRIPTION
**Pull request by marc-gr**
## What does this PR do?

Fixes a parsing issue for saml fileset. Was expecting an int as a string but the returned status codes are not numbers.

## Why is it important?

Fixes the NaN issue for saml fileset.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have made corresponding change to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

### CODE REVIEW

File: x-pack/filebeat/module/google_workspace/saml/config/pipeline.js [hunk 1]
```diff
@@ -32,14 +32,7 @@ var saml = (function () {
...
             // https://developers.google.com/admin-sdk/reports/v1/appendix/activity/saml
-            switch (p.name) {
-                case "status_code":
-                case "second_level_status_code":
-                    evt.Put("google_workspace.saml."+p.name, parseInt(p.value));
-                    break;
-                default:
-                    evt.Put("google_workspace.saml."+p.name, p.value);
-            }
+            evt.Put("google_workspace.saml."+p.name, p.value);
```
Comment (mtojek) [inline x-pack/filebeat/module/google_workspace/saml/config/pipeline.js:35]: I understand that it's not possible to have name == null :)

**Review by mtojek — APPROVED** [review]
LGTM

File: x-pack/filebeat/module/google_workspace/saml/config/pipeline.js (hunk 1 above)
Comment (marc-gr) [reply x-pack/filebeat/module/google_workspace/saml/config/pipeline.js:35]: Correct, name is always populated for parameters.

**Comment by Hanse00** [comment]
Thanks for addressing this 😃

**Pull request merged**